flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/core/testutils/FlinkCompletableFutureAssert.java

```diff
@@ -46,6 +48,9 @@
 
     private static final String SHOULD_HAVE_FAILED = "%nExpecting%n  <%s>%nto have failed.%n";
 
+    private static final String SHOULD_NOT_COMPLETED =
+            "%nExpecting%n  <%s>%nto have not completed normally or exceptionally.%n";
+
```

Comment:
I think this is not clearly enough when users encounter the exception because of lacking the timeout time. We'd better change it to the message as follows.
```suggestion
private static final String SHOULD_NOT_COMPLETED =
            "%nExpecting%n  <%s>%nto have not completed normally or exceptionally within %s ms.%n";
```